servers/account-data-deleter/src/batchDeleteHandler.ts

```diff
@@ -201,6 +210,25 @@ export class BatchDeleteHandler {
     let data: ReceiveMessageCommandOutput;
     let body: SqsMessage;
 
+    // Short-circuit if killswitch is on
+    // The unleash client is configured to check for new
+    // values every handful of seconds, so if this value
+    // is changed subsequent polls of the queue will pick it up
+    if (
+      this.unleashClient.isEnabled(
+        config.unleash.flags.deletesDisabled.name,
+        undefined,
+        config.unleash.flags.deletesDisabled.fallback,
+      )
+    ) {
+      // Schedule next poll and do nothing else
+      await this.scheduleNextPoll(
+        config.aws.sqs.accountDeleteQueue.defaultPollIntervalSeconds * 1000,
+      );
+      return;
```

Comment:
What if it was turned off, when does it pick up the turned back on? and run?

Reply:
Right now I have in the config to refresh every minute (handled by the unleash client)

Reply:
ahh